CODEOWNERS

```diff
@@ -93,6 +94,7 @@
 /gallery/workbook/Microsoft.Devices-IotHubs.json @microsoft/iotedge-workbooks
 /gallery/workbook/Microsoft.IoTCentral-IoTApps.json @microsoft/iotedge-workbooks
 /gallery/workbook/Azure*Security*Center.json @microsoft/azure-security-center-workbooks
+/gallery/workbook/defender-for-iot.json @microsoft/defender-for-iot-workbooks
```

Comment:
now that you've added it here it jumps out at me: is this workbooks on a place that's not a specific azure resource type, so you're using a custom "gallery" name?  (normally this would b e something liek the other ones, `microsoft.something/something`)  not necessarily a problem, just something that jumps out at me

Reply:
Hey,
We have our own resource provider - Microsoft.IoTSecurity, with a lot of resource types
Microsoft.IoTSecurity/alerts
Microsoft.IoTSecurity/devicves
Microsoft.IoTSecurity/recommendations
and a lot more. I thought of implementing it like ASC, who has a lot of resource types of their own as well.
Should I change it?

Reply:
I've left it as is, so we'll be able to support all resource types exists for our Extension.
Thanks!